server/src/main/java/org/opensearch/extensions/RegisterTransportActionsRequest.java

```diff
@@ -24,22 +24,22 @@
  */
 public class RegisterTransportActionsRequest extends TransportRequest {
     private String uniqueId;
-    private Map<String, Class> transportActions;
+    private Map<String, Class<?>> transportActions;
 
-    public RegisterTransportActionsRequest(String uniqueId, Map<String, Class> transportActions) {
+    public RegisterTransportActionsRequest(String uniqueId, Map<String, Class<?>> transportActions) {
```

Comment:
This is called from the SDK with this line:
```java
new RegisterTransportActionsRequest(uniqueId, transportActions),
```
See [comment here](https://github.com/opensearch-project/opensearch-sdk-java/pull/195#discussion_r996065662) for narrower types needed.